client/dashboard/plugins/plugin/sites-with-this-plugin.tsx

```diff
@@ -399,8 +404,45 @@ export const SitesWithThisPlugin = ( {
 						label: __( 'Delete' ),
 						modalHeader: getModalHeader( 'delete' ),
 						RenderModal: ( { items, closeModal } ) => {
-							const { mutateAsync } = useMutation( sitePluginRemoveMutation() );
-							const action = buildBulkSitesPluginAction( mutateAsync );
+							const { mutateAsync: deactivate } = useMutation( {
+								...sitePluginDeactivateMutation(),
+								onSuccess: () => {},
```

Comment:
Why do we need noop onSuccess function 🤔

Reply:
That's it, thanks ☺️